RSAKeyVaultProvider/RSAKeyVault.cs

```diff
@@ -134,5 +133,37 @@ public override byte[] EncryptValue(byte[] rgb)
         {
             throw new NotSupportedException();
         }
+#endif
+
+        private static HashAlgorithm GetHashAlgorithm(HashAlgorithmName algorithm)
+        {
+#if NETSTANDARD20
```

Comment:
Grrr, I'm hitting a bug when removing the `netstandard2.0` TFM. The following exception occurs while running the tests project on .NET 4.6.1:

> Could not load file or assembly 'System.Security.Cryptography.Algorithms, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.

I believe it's a manifestation of https://github.com/dotnet/corefx/issues/15570 that is a caused by a tooling issue in MSBuild: https://github.com/Microsoft/msbuild/issues/1346 :sweat_smile:

Not sure what to do... adding a manual binding redirect doesn't seem to be handy.